.github/workflows/test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
             ${{ runner.os }}-yarn-
 
       - name: yarn install
-        run: yarn --frozen-lockfile
+        run: yarn
```

Comment:
Might want to use the `--immutable` flag https://yarnpkg.com/cli/install#options-immutable

Reply:
thanks!